packages/terra-popup/src/Popup.jsx

```diff
@@ -135,6 +136,12 @@ class Popup extends React.Component {
     return true;
   }
 
+  handleRequestClose() {
+    const { targetRef } = this.props;
+    targetRef().focus();
+    this.props.onRequestClose();
+  }
+
```

Comment:
There needs to be a wdio test for this change ([example](https://github.com/cerner/terra-framework/blob/750799dcbe4aff7577e78b004d1d0356abb032b4/packages/terra-slide-panel/tests/wdio/slide-panel-spec.js#L169-L177)).